scalafmt-core/shared/src/main/scala/org/scalafmt/config/Docstrings.scala

```diff
@@ -79,6 +73,9 @@ object Docstrings {
   sealed abstract class Style {
     def skipFirstLine: Boolean
   }
+  case object Preserve extends Style {
+    override def skipFirstLine: Boolean = ???
```

Comment:
not important, but it may help in the future

```suggestion
    override def skipFirstLine: Boolean = throw new NotImplementedError("skipFirstLine called on Docstrings#Style#Preserve, it's a bug in scalafmt")
```

Reply:
will do